tools/server/server.cpp

```diff
@@ -2472,7 +2475,7 @@ struct server_context {
 
         res->truncated           = slot.truncated;
         res->n_decoded           = slot.n_decoded;
-        res->n_prompt_tokens     = slot.n_prompt_tokens;
+        res->n_prompt_tokens     = slot.n_prompt_tokens();
         res->n_tokens_cached     = slot.n_past;
```

Comment:
Is this still correct? Or should we have:

```suggestion
        res->n_tokens_cached     = slot.n_cache_tokens();
```